ql/src/test/queries/clientpositive/udf5.q

```diff
@@ -13,6 +13,14 @@ SELECT from_unixtime(unix_timestamp('2010-01-13 11:57:40', 'yyyy-MM-dd HH:mm:ss'
 
 SELECT from_unixtime(unix_timestamp('2010-01-13 11:57:40', 'yyyy-MM-dd HH:mm:ss'), 'MM/dd/yy HH:mm:ss'), from_unixtime(unix_timestamp('2010-01-13 11:57:40')) from dest1_n14;
 
+SELECT from_unixtime(unix_timestamp(cast('2010-01-13' as date)));
+
+SELECT from_unixtime(unix_timestamp('2010-01-13 11:57:40', 'yyyy-MM-dd HH:mm:ss'), 'MM/dd/yy HH:mm:ss');
+
+select from_unixtime(unix_timestamp('2010-01-13 11:57:40'), 'EEEE');
+
+select from_unixtime(unix_timestamp(cast('2010-01-13 11:57:40' || ' America/Los_Angeles' as timestamp with local time zone)) ,"yyyy-MM-dd'T'HH:mm:ssXXX") ;
+
```

Comment:
Can you add all the test cases listed in the JIRA description to confirm if we get matching output as Hive 1.2?

Reply:
Done